blocks/product-content-block/_index.scss

```diff
@@ -1,5 +1,14 @@
 @use "@wpmedia/arc-themes-components/scss";
 
 .b-product-content {
+	ul {
+		@include scss.block-properties("product-content-ul");
```

Comment:
Feels a little weird here calling these by the tag name.  Maybe `product-content-items` and later `product-content-item`?

Reply:
But it's not `items` we are allowing each tag to be controlled independently

Reply:
would `product-content-list` work here as a compromise?

Reply:
would `product-content-list` work here ?

Reply:
Oh, I see this is for styling the in-lined HTML... I suppose I'm be ok with `-list` and `-list-item`  It just feels dirty to me to use the tag names in the tokens.

Reply:
Just realised we do the same in the article body block for HTML elements that are set directly